packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDetailsSteps.tsx

```diff
@@ -231,6 +256,61 @@ export const TransactionsTableDetailsSteps = ({
         }
       />
 
+      {/* show mid transaction step for teleport tx */}
+      {isTeleportTx && (
```

Comment:
This JSX is very unreadable 😬 

I'd suggest splitting it into components so this is easier to read